app/scripts/views/mixins/modal-settings-panel-mixin.js

```diff
@@ -41,9 +41,11 @@ define(function (require, exports, module) {
       this.navigate('settings');
     },
 
-    onModalCancel () {
+    onModalCancel() {
```

Comment:
No biggie, but the other methods in this module all seem to have the space before parentheses.